freshmaker/utils.py

```diff
@@ -189,3 +190,21 @@ def _run_command(command, logger=None, rundir=None, output=subprocess.PIPE, erro
 def is_pkg_modular(nvr):
     """ Returns True if the package is modular, False otherwise. """
     return "module+" in nvr
+
+
+def get_ocp_release_date(ocp_version):
+    """ Get product release data via Product Page API
+
+    :param str ocp_version: ocp version
+    :return: date in format of "%Y-%m-%d", example: 2021-02-23
+    :rtype: str
+    """
+    if not conf.product_page_api_url:
+        raise RuntimeError("Product Page API url is not set in config")
+
+    ocp_release = f"openshift-{ocp_version}"
+
+    url = f"{conf.product_page_api_url.rstrip('/')}/releases/{ocp_release}/schedule-tasks"
```

Comment:
Optional: Could you use [urljoin](https://docs.python.org/3/library/urllib.parse.html#urllib.parse.urljoin)?

Reply:
`product_pages_api_url` (the base URL) contains the path to API, not just hostname, `urljoin` is used for get the full URL from a base URL and a relative URL, it can be easy to make mistake when we have path in baseurl and just want to concat the urls, so I'll just leave this as it is.

Reply:
@qixiang good point.